content/blogposts/2022/release-3.42

```diff
@@ -0,0 +1,55 @@
+---
+title: "Sourcegraph 3.42 release"
+publishDate: 2022-07-22T10:00-07:00
+description: "Sourcegraph 3.42 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.42"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+
+---
+
+Sourcegraph 3.42 is now available! For this release, we introduced:
+
+## All-new Admin Analytics 
+
+We believe providing a world-class admin-experience is critical to the future of Sourcegraph customers around the world. This starts with empowering administrators to measure and track the value of Sourcegraph within their organization.
+
+Historically, Sourcegraph administrators rely on word-of-mouth feedback from developers, leading to a poor understanding of the value of Sourcegraph within an organization. 
+
+In 3.42, we are releasing a completely redesigned user analytics experience for administrators. These visualizations will make it easier to understand developer engagement, and ultimately better manage user licenses, identify power users, and convey value to their internal leaders. 
+
+## Improved search-based navigation for Python non-locals
```

Comment:
@macraig What's the customer impact of this? Is it more accurate nav for certain use cases?

Reply:
Since this is a shorter update, I wonder if this may fit better as a changelog item. We also have a lot of short updates in the highlights this month.

Reply:
> Is it more accurate nav for certain use cases?
Yes, it's more accurate cross-file. It's already in the changelog so we can just remove it from this section if we already have a lot of updates.

Reply:
OK — Let's have this one be a changelog item for this month.